pymchelper/utils/mcscripter.py

```diff
@@ -117,14 +117,15 @@ def write(self):
                 logger.error('Symlink not created.')
                 print("Cannot create a link to the file, please check if you are using Linux or Python 3.")
                 raise e
-            except OSError as e:  # when python 2.7 is dropped this can be replaces tih FileExistsError: pass
+            except OSError as e:  # when python 2.7 is dropped this can be replaced with FileExistsError: pass
                 if e.errno == errno.EEXIST:
                     pass  # silently accept existing symlinks
                 else:
                     logger.error('Symlink not created.')
                     raise
         else:
             with open(self.path, 'w') as _f:
+                logger.info("Writing {}".format(self.path))
```

Comment:
@grzanka this produces no output, even when -v or -vvvv is given as opt.

I tried `logging.info("...")` as well. I recall you had a special way how you preferred to make this, for better comparability with the automated test, but cannot remember what you said?

Reply:
There is another caveat, `logging.basicConfig` should not be called twice as it is done now, once in `if __name__` block and another one in `main()` method.  According to https://docs.python.org/3/library/logging.html

> Does basic configuration for the logging system by creating a StreamHandler with a default Formatter and adding it to the root logger. The functions debug(), info(), warning(), error() and critical() will call basicConfig() automatically if no handlers are defined for the root logger.
> 
> **This function does nothing if the root logger already has handlers configured**, unless the keyword argument force is set to True.

I suggest putting default configuration (`logging.basicConfig()`) close to the `if args.verbosity` block, using `else` statement. Consider also using `elif` instead of second `if` there.

Reply:
As for `logging` vs `logger` : both can be used. I prefer for simplicity using simply `logging` (which mean using main root logger) in simple scripts with main is used.

Reply:
> There is another caveat, `logging.basicConfig` should not be called twice as it is done now, once in `if __name__` block and another one in `main()` method. According to https://docs.python.org/3/library/logging.html
> 
> > Does basic configuration for the logging system by creating a StreamHandler with a default Formatter and adding it to the root logger. The functions debug(), info(), warning(), error() and critical() will call basicConfig() automatically if no handlers are defined for the root logger.
> > **This function does nothing if the root logger already has handlers configured**, unless the keyword argument force is set to True.
> 
> I suggest putting default configuration (`logging.basicConfig()`) close to the `if args.verbosity` block, using `else` statement. Consider also using `elif` instead of second `if` there.

This fixed it, thanks. :)